Assets/__Scripts/UI/FirstBootMenu.cs

```diff
@@ -65,6 +66,9 @@ private void Start()
         directoryCanvas.SetActive(true);
     }
 
+    public void InstallAdb() => StartCoroutine(AdbUI.DoDownload());
```

Comment:
If the dialog box inside `AdbUI.DoDownload()` is unclosable by design, this should _really_ be wrapped in another "Are you sure" box, or offer the ability to cancel the ADB download.

I'm not going to have people click this button (either unintentionally, out of curiosity, etc.) and have no effective way to prevent or cancel the download without closing ChroMapper entirely. It would also be smart to expose UI for uninstalling ADB. Not all ChroMapper users are Quest users.

Reply:
Realistically they only lost a few MB of storage (30MB on Windows) and it runs ADB server in the background when editing song metadata. Wouldn't say PC users would even notice or care unless you tell them. I was planning on adding uninstall anyways somehow, though I'm probably too lazy to add a cancel method or "are you sure " dialog since that means more spaghetti for me. The download is also about 10MB which for majority of users is too small to even cancel out of.